ts/IterableInboxMessageDisplay.tsx

```diff
@@ -46,13 +49,40 @@ const IterableInboxMessageDisplay = ({
    returnButtonContainer = {...returnButtonContainer, marginTop: Platform.OS === 'android' ? 0 : 40}
    returnButtonContainer = (!isPortrait) ? {...returnButtonContainer, marginTop: 10} : returnButtonContainer
 
+   let JS = `
+      const links = document.querySelectorAll('a')
+      links.forEach(link => {
+         link.addEventListener("click", () => {
+            window.ReactNativeWebView.postMessage(link.href)
+         })
+      })
+   `
+
+   let action = new IterableAction("", "", "")
+   let source = IterableActionSource.inApp
+   let context = new IterableActionContext(action, source)
+
    useEffect(() => {
       inAppContentPromise.then(
          (value) => {
             setInAppContent(value)
          })
    })
 
+   const handleHTMLMessage = (event: any) => {
+      if(event.nativeEvent.data === 'iterable://delete') {
+         deleteRow(rowViewModel.inAppMessage.messageId)
+         returnToInbox()
+      } else if(event.nativeEvent.data === 'iterable://dismiss') {
+         returnToInbox()
+      } else {
+         if(Iterable.savedConfig.urlHandler) {
+            Iterable.savedConfig.urlHandler(event.nativeEvent.data, context)
```

Comment:
This takes care of the deep link where urlHandler is called. But it still wont takes care of scenario where they return false and SDK opens the url in browser.

Reply:
Yes, I plan to add the external linking on a separate PR today.

Reply:
sounds good 👌